discovery_utils/utils/keywords.py

```diff
@@ -0,0 +1,212 @@
+"""
+keyword_utils.py
```

Comment:
```suggestion
discovery_utils.utils.keywords
```

file isn't named keyword_utils.py, and when this docstring is shown in IDE, it would be nice to know the relative/ module path.

Reply:
Note: it would be nice in the future when time permits, _**where script paths are included**_, to update docstrings to show the **module path** _relative to_ the **package root** as in this example.